### ISSUE TITLE
TelDrive not starting after 1.3.9

### ISSUE THREAD

**Issue by Crypt0x69** [divyam234/teldrive#271]
Did anything changed after 1.3.9 ?

Version 1.3.9 was the last working version.

System: Linux ubuntu 5.15.0-107-generic #117-Ubuntu SMP Fri Apr 26 12:26:49 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

Teldrive closes without any messages.

Tested on X64 and ARM (PI3).

Maybe it's importent: I'am using Telegramm Premium

Edit: with an empty & new database it's starting again! whats going on here?

Maybe this commit? > https://github.com/divyam234/teldrive/commit/d0d0f2f8ed4dc78ac88e4f3084ce9a0e44a854d0

**Comment by Linden10** [comment]
Ah the reason teldrive 1.3.10 isn't working is because of the fix that disallows duplicate folders and that when you start teldrive 1.3.10, it's trying to add unique files and folders to the database (files not having the same filename within the same folder and folders not having the same filename within a folder) but failing because they're duplicate folders of some kind in one of your folders.

I had trouble with this myself since I noticed that when I uploaded a bunch of files in the last few teldrive versions, it was creating duplicate folders accidently...

Anyways I was able to fix and upgrade to 1.3.10 with this sql command that you can input into your postgresql/neondb/etc editor!

```
SELECT name, parent_id, COUNT(*)
FROM teldrive.files
WHERE type = 'folder'::text
GROUP BY name, parent_id
HAVING COUNT(*) > 1;
```

Enter this command to list duplicate folders with the same parent_id (parent_id being the folder that the folders reside in), you can then use said parent_id or name to browse said folder and either rename, move or delete said folder.

If you're still having trouble finding the folder, enter the parent_id with your teldrive web ui to open up said folder:
http://your_teldrive_localurl/browse?parentId=ENTER_PARENTID_HERE

Hope this helps ya out! Be sure to check any duplicate folders for important data if you decide to delete it, otherwise just rename or move it somewhere else where they're no other duplicates to be found!

**Comment by Crypt0x69** [comment]
Thank you so much.

Worked for me.
